scripts/common/rook.sh

```diff
@@ -246,3 +246,33 @@ function rook_operator_ready() {
     fi
     return 0
 }
+
+function rook_is_healthy_to_upgrade() {
+    log "Awaiting Rook Ceph health ..."
+    if ! $DIR/bin/kurl rook wait-for-health 600 ; then
+        kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
+        logFail "Rook Ceph is unhealthy"
+        return 1
+    fi
+
+    log "Checking Rook versions and replicas"
```

Comment:
```suggestion
    log "Checking Rook Ceph versions and replicas"
```

Reply:
You do not check replicas. I fear the logic will get very complex in bash. Could you use the go command `wait-for-version` or add another go command `verify-version`?

Reply:
I do not think so, see: https://github.com/replicatedhq/kURL/pull/4201#discussion_r1137599181

Reply:
ok. i will approve since this is an improvement